android/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="splash_bg">#ffffff</color>
+    <color name="primary_dark">#ffffff</color>
```

Comment:
This `primary_dark` colour is required due to a bug in `react-native-splash-screen`